openmrs-orthanc/src/main/resources/config/application.properties

```diff
@@ -0,0 +1,189 @@
+
+# Path to file where offsets are to be stored, maps to property named database.history.file.filename, DO NOT change
+# after setting it
+debezium.historyFilename=${DEBEZIUM_HISTORY_FILENAME:.history.txt}
+
+# Maximum size of each batch of source records to be read by debezium from the binlog files, maps to the
+## debezium MySQL connector property named max.batch.size
+#https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-property-max-batch-size
+debezium.reader.maxBatchSize=${DEBEZIUM_READER_MAX_BATCH_SIZE:2048}
+
+# Specifies the criteria for running a snapshot when the connector starts, DO NOT change after setting it, maps to the
+# debezium MySQL connector property named snapshot.mode
+debezium.snapshotMode=schema_only
+
+debezium.snapshotLockingMode=extended
+# ----------------------------------------------------------------------------------------------------------------------
+
+# *********************** OAuth2 Configuration ********************************************************************
+# Enabled Oauth when set to true, defaults to false.
+oauth.enabled=${OAUTH_ENABLED:false}
+
+# The client Id of the account, defaults to empty.
+oauth.access.token.uri=${OAUTH_ACCESS_TOKEN_URL:}
+
+# The client Id of the account to use to authenticate, defaults to empty.
+oauth.client.id=${OAUTH_CLIENT_ID:}
+
+# The client secret of the account to use to authenticate, defaults to empty.
+oauth.client.secret=${OAUTH_CLIENT_SECRET:}
+
+# Authentication scope, can be multiple values separated by commas, defaults to empty.
+oauth.client.scope=${OAUTH_CLIENT_SCOPE:}
+# ----------------------------------------------------------------------------------------------------------------------
+
+# *********************** Orthanc configuration **************************************************************************
+#
+# The base URL to the Orthanc instance
+orthanc.baseUrl=${ORTHANC_URL:http://orthanc:8042}
+
+# The database name to use to log into the Orthanc instance
+orthanc.database=${ORTHANC_DATABASE:orthanc}
+
+# The username for the account to use to log into the Orthanc instance
+orthanc.username=${ORTHANC_USER:orthanc}
+
+# The password for the account to use to log into the Orthanc instance
+orthanc.password=${ORTHANC_PASSWORD:orthanc}
+
+# Whether to enable the patient sync feature. If enabled, patient data will be synchronized between OpenMRS and Orthanc.
+orthanc.openmrs.enable.patient.sync=${ORTHANC_OPENMRS_ENABLE_PATIENT_SYNC:true}
+# ----------------------------------------------------------------------------------------------------------------------
+
+# *********************** OpenMRS configuration **************************************************************************
+#
+# The username for the account to use to log into the OpenMRS instance
+openmrs.username=${OPENMRS_USERNAME:admin}
+
+# The password for the account to use to log into the OpenMRS instance
+openmrs.password=${OPENMRS_PASSWORD:Admin123}
```

Comment:
These properties will always be same as `eip.fhir.username` and `eip.fhir.password` respectively, can we find to somehow reuse them?

Reply:
I feel it becomes confusing if we are using OpenMRS REST APIs but using `eip.fhir.*` credentials. That's why I wanted to keep these separate.